### ISSUE TITLE
[Bug]: StackedBarChart callout (tooltip) info is inaccurate when hovering legend area

### ISSUE THREAD

**Issue by DVDripXvid** [microsoft/fluentui#22507]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
    Memory: 7.32 GB / 15.88 GB
  Browsers:
    Chrome: 100.0.4896.88
    Edge: Spartan (44.19041.1266.0), Chromium (100.0.1185.39)
    Internet Explorer: 11.0.19041.1566
  npmPackages:
    @fluentui/react: ^8.64.0 => 8.64.0
    @fluentui/react-charting: ^5.9.16 => 5.9.16
```

### Reproduction

https://codepen.io/dvdripxvid/pen/OJzBMzp

## Actual Behavior

Affected component: @fluentui/react-charting - StackedBarChart

Moving the cursor from the chart area over the legends, the callout (tooltip) is still shown but provides partially inaccurate info:
 - Label: good (shows the current info based on mouse hovering)
 - Value: does not update regardless of what item you hovering
 - Color: does not update regardless of what item you hovering

## Expected Behavior
The callout should provide accurate info based the selected(hovered) legend item.

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by AtishayMsft** [comment]
Thanks for reporting. We will take a look into this issue.

**Comment by DVDripXvid** [comment]
I think this bug is still not fixed.

**Comment by AtishayMsft** [comment]
This issue has been fixed with PR #25978
